lib/portlayer/event/manager.go

```diff
@@ -18,11 +18,11 @@ import (
 	"fmt"
 	"sync"
 
-	log "github.com/Sirupsen/logrus"
-
 	"github.com/vmware/vic/lib/portlayer/event/collector"
 	"github.com/vmware/vic/lib/portlayer/event/events"
 	"github.com/vmware/vic/pkg/trace"
+
+	log "github.com/Sirupsen/logrus"
```

Comment:
just curious, why this import is moved?

Reply:
during an iteration of the development it wasn't needed, but had to revert back to using it...apparently I changed the location...